ansible_navigator/image_manager/image_inspector.py

```diff
@@ -0,0 +1,88 @@
+""" inspect all images """
```

Comment:
The name of the file can be renamed to `image_manager/inspect.py`? Since `image_` prefix is present in the parent folder not need to repeat in the file IMO.

Reply:
yes.  this is something I might say :)

Reply:
yes